activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceControllerImplIT.java

```diff
@@ -124,22 +123,28 @@ public void setUp() {
         assertThat(processEngineChannels).isNotNull();
         assertThat(processDeployedProducer).isNotNull();
     }
-
+    
     @Test
     public void getProcessInstances() throws Exception {
-
+        //given
         List<ProcessInstance> processInstanceList = Collections.singletonList(defaultProcessInstance());
-        Page<ProcessInstance> processInstances = new PageImpl<>(processInstanceList,
-                                                                processInstanceList.size());
-        when(processRuntime.processInstances(any())).thenReturn(processInstances);
-
-        this.mockMvc.perform(get("/v1/process-instances"))
+        Page<ProcessInstance> processInstancePage = new PageImpl<>(processInstanceList,
+                                                                   processInstanceList.size());
+        
+        when(processRuntime.processInstances(any())).thenReturn(processInstancePage);
+                
+        //when
+        mockMvc.perform(get("/v1/process-instances?skipCount=10&maxItems=10")
+                .accept(MediaType.APPLICATION_JSON))
+                //then
+                .andDo(print())
                 .andExpect(status().isOk())
                 .andDo(document(DOCUMENTATION_IDENTIFIER + "/list",
-                                responseFields(subsectionWithPath("page").description("Pagination details."),
-                                               subsectionWithPath("_links").description("The hypermedia links."),
-                                               subsectionWithPath("_embedded").description("The process definitions."))));
-    }
+                                pageRequestParameters(),
+                                pagedResourcesResponseFields()
```

Comment:
You need to use [pagedProcessInstanceFields](https://github.com/Activiti/activiti-cloud-service-common/blob/elias-fix-hal-collection-rel-name/activiti-cloud-services-rest-docs/src/main/java/org/activiti/alfresco/rest/docs/HALDocumentation.java#L128) here. Note that it's not yet merged in develop and you need to rely on snapshots locally.

Reply:
Have made this, but then a test failed...